index.bs

```diff
@@ -15,19 +15,107 @@ Complain About: accidental-2119 yes, missing-example-ids yes
 Markup Shorthands: markdown yes, css no
 Assume Explicit For: yes
 </pre>
+<pre class="anchors">
+spec: HTTP; urlPrefix: https://httpwg.org/specs/rfc7231.html#
+    type: dfn; text: HTTP 3xx statuses; url: status.3xx
+spec: tracking-dnt; urlPrefix: https://www.w3.org/TR/tracking-dnt/#
+    type: dfn; text: tracking; url: dfn-tracking
+</pre>
 
 <section class="non-normative">
 <h2 id="intro">Introduction</h2>
 
 <em>This section is non-normative.</em>
 
-Blah blah blah.
+Browsers are working to prevent cross-site [=tracking=], which threatens user
+privacy. In addition to third-party cookies and storage, other client-side
+methods exist that enable cross-site tracking. [=Navigational tracking=]
+correlates user identities across sites during navigations between those sites.
+[=Navigational tracking=] uses [=link decoration=] to convey information, but
+not all [=link decoration=] is tracking. This project attempts to distinguish
+tracking from non-tracking navigation and to prevent the tracking without
+damaging similar but benign navigations.
+
 </section>
 
 <h2 id="infra">Infrastructure</h2>
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+# Terminology # {#terminology}
+
+<dfn>Link decoration</dfn> is when the source of a [=hyperlink=] "decorates" its [=URL=]
+with extra information beyond what's necessary to identify the page a user wants
+to navigate to. This information can be placed almost anywhere inside the URL.
+
+<dfn>Navigational tracking</dfn> refers to the general use of a
+[[HTML#browsing-the-web|navigation]] to identify that a user on one site is the
+same person as a user on another site. Navigations are usually visible to users,
+but it's possible for a site to navigate away and redirect back fast enough that
+only attentive users would notice.
```

Comment:
This definition sounds a bit like navigational tracking always employs fast navigations for tracking (like bounce tracking), though IIUC we're looking at more than that. Maybe "general use of a navigation or the information that is transmitted across sites as part of a navigation"?

Reply:
Thanks, that seems good!